Jarvis-Platform/jarvis-studio
# Feature/launch dag

[comment]: [![Status](https://img.shields.io/badge/Status-DEVELOPMENT-orange.svg)]()
[comment]: [![Status](https://img.shields.io/badge/Status-HOLD-inactive.svg)]()

[![Status](https://img.shields.io/badge/Status-READY-brightgreen.svg)]()


## Description
Ad the possibility to launch a dag from studio.


## Todos
- [x] Add launch item in list
- [x] Add confirmation modal
- [x] Use cloud function


## Deploy Notes
Review & merge


## Steps to Test or Reproduce
```sh
git checkout master
git pull
git checkout feature/launch-dag
```

1. Go to a configuration view
2. Launch a dag
3. Wait for about 3min => You should see it running


## Impacted Areas in Application
- `src/components/data-workflows/common/item/overview/ViewHeader.vue`
